src/ashley/admin.py

```diff
@@ -0,0 +1,9 @@
+"""Admin of the Ashley application."""
+from django.contrib import admin
+
+from .models import AshleyUser
```

Comment:
@madmatah you sure :stuck_out_tongue:?

Reply:
haha good catch, I just saw that while moving AshleyUser in the sandbox :smile: